types/index.ts

```diff
@@ -37,9 +37,8 @@ export type Theme = {
 }
 export interface User {
   id: string
-  firstName: string
-  lastName: string
   username: string
+  fullname: string
```

Comment:
The backend sends the property of "name" not "fullname", could you adapt to this as well?